activiti-cloud-starter-configure/src/main/java/org/activiti/spring/boot/AbstractProcessEngineAutoConfiguration.java

```diff
@@ -117,6 +118,10 @@ protected SpringProcessEngineConfiguration baseSpringProcessEngineConfiguration(
       conf.setCustomMybatisXMLMappers(new HashSet<String>(activitiProperties.getCustomMybatisXMLMappers()));
     }
     
+    if (activitiProperties.isUseStrongUuids()) {
```

Comment:
It's not mandatory, but is there any chance to get unit tests covering this configuration? I mean, by mocking ActivititiProperties and other necessary beans.